samples/tlssurvey/build.gradle.kts

```diff
@@ -10,7 +10,7 @@ application {
 
 dependencies {
   implementation("com.squareup.okhttp3:okhttp:5.0.0-alpha.10")
-  implementation("com.squareup.okhttp3:okhttp-coroutines:5.0.0-alpha.10")
+  implementation("com.squareup.okhttp3:okhttp-coroutines:5.0.0-alpha.11")
```

Comment:
I don’t like that it’s upgraded `okhttp-coroutines` without `okhttp` standalone. That’ll hopefully come in a follow-up.

(This is more interesting for other users of OkHttp, who will suffer weird crashes when versions don’t match up.)